app/controllers/acounts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AcountsController < ApplicationController
+
+  def new
+  	@todoist = Todoist.new
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Style/Tab: Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)